src/style.css

```diff
@@ -0,0 +1,67 @@
+body {
+  margin: 0;
+}
+
+.star {
+  display: flex;
+  width: 96px;
+  justify-content: space-between;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

Comment:
```
.stars__star:not(:last-child) {
  margin-right: 4px;
}
``` 
such selector set margin for each star in row except last.
it will make proper width of stars and can help to fix some tests.